src/index.tsx

```diff
@@ -4,356 +4,11 @@
  * SPDX-License-Identifier: ISC
  */
```

Comment:
Please retain the license header across files

Reply:
done